terraform/meta/bootstrap-env/space.tf

```diff
@@ -0,0 +1,35 @@
+# We need the Org GUID to describe spaces
+data "cloudfoundry_org" "org" {
+  name = var.org_name
+}
+
+data "cloudfoundry_asg" "asgs" {
+  for_each = toset(var.asgs)
+  name     = each.key
+}
+
+# Ensure the space exists and is configured as expected
+resource "cloudfoundry_space" "space" {
+  name = var.name
+  org  = data.cloudfoundry_org.org.id
+
+  # Disallow SSH access for the special name production
+  allow_ssh = var.name != "production" && var.name != "production-egress"
```

Comment:
This could be a parameter of the space (e.g. instead of looking for the space name).